src/platforms/web/server.js

```diff
@@ -1,12 +1,74 @@
+const { readFileSync, writeFileSync } = require('fs');
 const express = require('express');
+const fetch = require('node-fetch');
 const path = require('path');
 const app = express();
 const port = 1337;
+const headRegex = /(<head>).*(<\/head>)/g;
 
 app.use(express.static(__dirname));
 
-app.get('*', function(req, res) {
-  res.sendFile(path.join(__dirname, '/index.html'));
+app.get('*', async(req, res) => {
+  let finalHtml = '';
+  let identifier = req.url.split('/').pop();
+  let isAppPage = req.url.includes('/$/');
+  let isChannel = req.url.includes('/@');
```

Comment:
This test probably needs to be stronger, as I think `@` can technically appear in the middle of a claim name

Reply:
@ is a reversed character.